### PR TITLE
Scala-steward PR mashup 5640 5692

### PR DESCRIPTION
**Pull request by cjllanwarne**
Possibly a little optimistic...

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -120,14 +125,14 @@ object Dependencies {
...
-    "literal"
-  ).map(m => "io.circe" %% s"circe-$m" % circeV) :+ circeYamlDependency
+    "io.circe" %% s"circe-core" % circeV,
+    "io.circe" %% s"circe-parser" % circeV,
+    "io.circe" %% s"circe-generic" % circeV,
+    "io.circe" %% s"circe-generic-extras" % circeV,
+    "io.circe" %% s"circe-shapes" % circeV,
+    "io.circe" %% s"circe-refined" % circeV,
+    "io.circe" %% s"circe-literal" % circeV
+  ) :+ circeYamlDependency
```
Comment (mcovarr) [inline project/Dependencies.scala:135]: ooc was this de-DRYed because it confused Steward?

Comment (mcovarr) [reply project/Dependencies.scala:135]: Judging by the titles I believe the intent of Scala Steward PRs #5640 and #5641 was to:

1. ) Upgrade all of these dependencies except circe-generic-extras to 0.12.3
2. ) Upgrade circe-generic-extras to 0.12.2

Those aren't the code changes the Scala Steward PRs actually produce (all of these dependencies flip from 0.12.1 to 0.12.3 to 0.12.2), but that's another matter. 🙂 Here's [what I did](https://github.com/broadinstitute/cromwell/pull/5711/commits/7f28d1a9741b30f12811366648082f92a5634937) in my bite-size #5711 PR.

File: project/Dependencies.scala [hunk 2]
```diff
@@ -43,51 +43,51 @@ object Dependencies {
...
-  private val hsqldbV = "2.4.1"
+  private val hsqldbV = "2.5.1"
   private val http4sVersion = "0.20.0-M5"
   private val jacksonV = "2.10.5"
-  private val jacksonJqV = "1.0.0-preview.20190925"
-  private val janinoV = "3.0.12"
+  private val jacksonJqV = "1.0.0-preview.20191208"
+  private val janinoV = "3.0.16"
   private val javaxActivationV = "1.2.0"
   private val jaxbV = "2.3.2"
```
Comment (mcovarr) [inline project/Dependencies.scala:55]: This is not upgrading jaxbV to 2.3.3 per #5632 or #5663 nor providing a Scala Steward ignore directive.

Comment (cjllanwarne) [reply project/Dependencies.scala:55]: Correct. I was hoping we could circle back to this independently after the majority of the updates were merged

File: project/Dependencies.scala (hunk 1 above)
Comment (cjllanwarne) [reply project/Dependencies.scala:135]: Yeah, our ad-hoc dependency name generation seemed to be causing scala-steward some issues with version tracking and concurrent-updates-to-different-versions, so I tried to make it easier for it.

**Comment by cjllanwarne** [comment]
Closing in favor of @mcovarr 's 10-by-10 approach

**Pull request closed without merging**